tests/queries/0_stateless/03290_dictionary_assert_on_function.sql

```diff
@@ -0,0 +1,15 @@
+CREATE DICTIONARY default.currency_conversion_dict
+(
+    `a` String,
+    `b` Decimal(18, 8)
+)
+PRIMARY KEY a
+SOURCE(CLICKHOUSE(
+    TABLE ''
+    STRUCTURE (
+        a String
+        b Decimal(18, 8)
+    )
+))
+LIFETIME (MIN 0 MAX 3600)
+LAYOUT (FLAT()); -- {serverError INCORRECT_DICTIONARY_DEFINITION}
```

Comment:
I don't quite understand why this definition is incorrect ...

Reply:
It doesn't have such table, I've marked the table as empty: ''